diff --git a/BUILD.bazel b/BUILD.bazel
new file mode 100644
index 0000000..098b885
--- /dev/null
+++ b/BUILD.bazel
@@ -0,0 +1,20 @@
+
+cc_library(
+    name = "tinyxml2",
+    srcs = ["tinyxml2.cpp"],
+    hdrs = ["tinyxml2.h"],
+    visibility = ["//visibility:public"],
+)
+
+cc_test(
+    name = "xmltest",
+    srcs = ["xmltest.cpp"],
+    size = "small",
+    data = 
+        glob(["resources/*.xml"]) + # Input files
+        [ "resources/out/readme.txt" ], # Hack to create output dir
+    visibility = ["//visibility:private"],
+    deps = [
+        ":tinyxml2"
+    ]
+)
